src/mbgl/gl/context.cpp

```diff
@@ -838,6 +838,9 @@ void Context::performCleanup() {
     MLN_TRACE_FUNC();
 
 #ifndef NDEBUG
+    // In debug builds, un-bind all texture units so that any incorrect use results in an
+    // error rather than using whatever texture happened to have been bound previously.
+    // This takes some time, however, so don't do it in release builds.
     // TODO: Find a better way to unbind VAOs after we're done with them without
```

Comment:
Is this TODO still relevant?

Reply:
I don't really know.  Assuming it means better in the sense of performance, so long as it's only in the debug build there's no need to improve it.

Reply:
Looks like it goes back pre-fork, so whatever.